recipes/cpython/all/conanfile.py

```diff
@@ -148,12 +155,12 @@ def validate(self):
             if self.settings.build_type == "Debug" and "d" not in self.settings.compiler.runtime:
                 raise ConanInvalidConfiguration("Building debug cpython requires a debug runtime (Debug cpython requires _CrtReportMode symbol, which only debug runtimes define)")
             if self._is_py2:
-                if self.settings.compiler.version >= tools.Version("14"):
+                if self.settings.compiler.version >= Version("14"):
```

Comment:
Would it be possible to use `check_min_vs()` here so that it works with the newer `msvc` compiler?

Reply:
These are changes suggested by @uilianries , I'll let him handle them.

Note that this PR initially only suggested a one line change, as specified in this commit: https://github.com/conan-io/conan-center-index/pull/9891/commits/d6103f2faaab4f4b92ca32519195872502043e37

Reply:
`check_min_vs()` raises `ConanInvalidConfiguration`. Here, we just want an warning message. Docs: https://docs.conan.io/en/latest/reference/conanfile/tools/microsoft.html#check-min-vs